pom.xml

```diff
@@ -3,45 +3,54 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>2.36</version>
+    <version>3.48</version>
   </parent>
 
   <artifactId>authorize-project</artifactId>
   <version>1.3.1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Authorize Project</name>
   <description>Configure projects to run with specified authorization.</description>
+  <url>https://wiki.jenkins.io/display/JENKINS/Authorize+Project+plugin</url>
 
-  <url>https://wiki.jenkins-ci.org/display/JENKINS/Authorize+Project+plugin</url>
-  
   <scm>
     <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
     <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
     <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
     <tag>HEAD</tag>
   </scm>
-  
+
   <developers>
     <developer>
       <id>ikedam</id>
       <name>IKEDA Yasuyuki</name>
     </developer>
   </developers>
-  
+
   <licenses>
     <license>
       <name>The MIT license</name>
       <url>http://www.opensource.org/licenses/mit-license.php</url>
       <distribution>repo</distribution>
     </license>
   </licenses>
-  
+
   <properties>
-    <jenkins.version>1.625</jenkins.version><!-- for JENKINS-28298 -->
-    <java.level>7</java.level>
-    <workflow.version>1.0</workflow.version> <!-- only for testing purpose -->
+    <jenkins.version>2.60.3</jenkins.version>
+    <java.level>8</java.level>
```

Comment:
I believe you don't need this change for this request.
It would be a part of changes for the upcoming JCaC change.
Would you revert this change to clarify what is actually required to have tests run against latest cores?

Reply:
And please keep the comment "only for testing purpose".

Reply:
I reverted the removal of the "only for testing purpose" comment.

Reply:
I see your point in just wanting to have the minimum required changes. But not performing the Java and Jenkins version change from commit f65754e7c9952adce3aab5ddded5f6ef726e35a1 "Modernize plugin (upgrade parent pom & Java 8 compatibility)" means ripping it apart.

That's something I don't want to do. It would require more effort now and complicates things.

For example:
- The parent "plugin-pom" dropped support for Jenkins 1.x in version 3.x (see https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#30).
- Not upgrading Jenkins also means additional entries for `<dependencyManagement>` to satisfy the maven-enforcer-plugin. They should then later be reverted when the upgrade is performed.

I still suggest to first merge #37 standalone and perform all other changes afterwards.
This pull request is currently setup in a way that it exactly contains all commits of #37 (commit 1-4) and additionally adds commits to fix tests for newer Jenkins versions (commit 5-7).

BTW I performed changes similar to #37 for many other plugins over the last year.

Reply:
I got your point. Let me have some more time to think over that.

Reply:
This is consistent with https://github.com/jenkinsci/plugin-pom/pull/83/files#r146937116 .
I accept it.